TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/opmode/SplineTest.java

```diff
@@ -30,7 +30,7 @@ public void runOpMode() throws InterruptedException {
         sleep(2000);
 
         drive.followTrajectory(
-                drive.trajectoryBuilder(new Pose2d(30, 30, Math.toRadians(180)), true)
+                drive.trajectoryBuilder(new Pose2d(30, 30, Math.toRadians(0)), true)
```

Comment:
I'd prefer `traj.end()` instead. One of 0.5.1's goals was to fix this old idiom.